package.json

```diff
@@ -2,20 +2,28 @@
   "name": "team-green-v2-app",
   "version": "0.1.0",
   "private": true,
+  "scripts": {
+    "clean:yarn-cache": "yarn cache clean",
+    "clean:node_modules": "rmdir node_modules /s",
+    "lint:fix": "eslint \"src/**/*.{js,jsx,ts,tsx}\" --fix",
+    "prettier:format": "prettier --write \"src/**/*.{js,jsx,ts,tsx}\"",
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
   "dependencies": {
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
-    "@testing-library/user-event": "^13.5.0",
-    "react": "^18.1.0",
-    "react-dom": "^18.1.0",
+    "@testing-library/user-event": "^14.2.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
+    "styled-components": "^5.3.5",
     "web-vitals": "^2.1.4"
   },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+  "devDependencies": {
+    "@types/styled-components": "^5.1.25"
```

Comment:
Jeśli nie będziemy używać TypeScriptu to nie będzie potrzebne

Reply:
Nie ruszałem dotąd ts, ale po krótce zapoznawszy się z tym artykułem https://frontlive.pl/blog/typescript-podstawy 
Pomyślałem że nawet używanie typów w parametrach i wartości zwracanej jest już pomocne.
Co sądzisz na temat takiego wprowadzania ts w projekcie?

Reply:
Generalnie ustaliliśmy, że:
1. Osoby, które chcą zmierzyć się z TypeScriptem mogą pisać w TS
2. Osoby, które mają mniej doświadczenia będą pisać w czystym JS swoje komponenty. Konfiguracja TS ma być taka by przepuszczać JS.
3. Na PR warto zaznaczać czy wybieramy TS czy JS dla naszego fragmentu kodu
4. Zgadzamy się na taką niespójność (raz TS raz JS) by "nie było za dużo na raz". Mamy osoby na różnym poziomie znajomości JS/TS więc nie chcemy by nikt utknął.
5. TS jak najbardziej warto znać więc zachęcam do TS (choć sam go nie lubię :) )
6. Co do tego co typujemy: Otwarta dyskusja - miej tylko na uwadze, że nie wszystko warto typować - TS ma stara się wydedukować typy
7. Zerknij na Slacka bo tam chyba @sliwia wrzucała konfigurację TS ale nie miała możliwości pushowania do githuba. Ja zerknę na to jutro wieczorem ostatnio byłem całkiem zajęty "prozą życia" :)

Reply:
Ok, zerknąłem, Patryk dodał nas już do repo.